server/ServerConfig.ts

```diff
@@ -19,6 +19,7 @@ export interface ServerConfig {
       ENABLE_ACCOUNT_REGISTRATION_ACCEPT_TERMS_AND_PRIVACY_POLICY: boolean;
       ENABLE_DEBUG: boolean;
       ENABLE_DOMAIN_DISCOVERY: boolean;
+      ENABLE_FEDERATION: boolean;
```

Comment:
At some point it might be interesting to have a separate section for `FEDERATION` outside of `FEATURE`.

kinda:
```
FEDERATION: {
  ENABLE_DOMAIN_DISCOVERY: boolean,
  FEDERATION_DOMAIN: string,
}
```

Reply:
Yes, I am prefix all federation-related keys already with `FEDERATION_` (except `ENABLE_FEDERATION`) to identify these.